### PR TITLE
[master][Doc change] - replace pip command to a working version

### PR DESCRIPTION
**Pull request by ZsBT**
To suggest using salt-pip instead of the native pip.

### CODE REVIEW

File: doc/topics/cloud/vmware.rst [hunk 1]
```diff
@@ -19,12 +19,11 @@ available at PyPI:
...
 https://pypi.org/project/pyvmomi/
 
-This package can be installed using `pip` or `easy_install`:
+This python module can be installed in the onedir package:
 
 .. code-block:: bash
 
-    pip install pyvmomi
-    easy_install pyvmomi
+    salt-pip install pyVmomi
```
Comment (dmurphy18) [inline doc/topics/cloud/vmware.rst:26]: Why the capital V, in PyPI is it pyvmomi

**Review by dmurphy18 — CHANGES_REQUESTED** [review]

File: doc/topics/cloud/vmware.rst (hunk 1 above)
Comment (dmurphy18) [reply doc/topics/cloud/vmware.rst:26]: Just being picky since pyVmomi works too, but a pip3 list will show pyvmomi, hence wanting the correction

Comment (ZsBT) [reply doc/topics/cloud/vmware.rst]: Yes you're absolutely right, the proper name is in lowercase. Let me give it another try in a new PR

**Comment by ZsBT** [comment]
cancelling PR due to improper package name

File: doc/topics/cloud/vmware.rst (hunk 1 above)
Comment (dmurphy18) [reply doc/topics/cloud/vmware.rst]: This PR is fine looking now

**Comment by dmurphy18** [comment]
@ZsBT I think you need to read up on workflows with GitHub and using Git.  FYI there was no reason to closed this PR and create a new PR for that change, you should have merely updated, committed and pushed up your changes. Closing and creating new PRs kills the history of the original PR and any comments or reviews made:

Example commands, after ensuring ```pre-commit install``` on your branch:

edit doc/topics/cloud/vmware.rst
git add doc/topics/cloud/vmware.rst
git commit
git push -u -f origin ZsBTpatch-1 (or whatever your branch name is)

**Pull request closed without merging**